components/buttons/BlueButton.tsx

```diff
@@ -1,23 +1,52 @@
 import { Button, ButtonProps } from '@mui/material';
```

Comment:
I found an error when trying to install packages.
Please do a backup. Delete "node_modules" 
We need to solve it because it does not work using a different machine
<img width="812" alt="Captura de pantalla 2024-12-11 a la(s) 9 28 01 a  m" src="https://github.com/user-attachments/assets/de188bc4-e1dc-4af3-ac52-f2a493cd6213">
folder and run `npm install`